components/list/list-item.js

```diff
@@ -182,6 +198,17 @@ class ListItem extends RtlMixin(LitElement) {
 	render() {
 		const contentId = getUniqueId();
 		const checkBoxId = getUniqueId();
+		const link = html`
+			<a class="d2l-list-item-link"
```

Comment:
We could potentially move this to a render helper on the LinkMixin - the only think that is on here that's not a property of the mixin is the class.  ex.
```
renderLink(inner) {
   return html`<a ...etc>${inner}</a>`;
}
```
In the case of the button variants, we repeated it, but there's possibly some refactoring we could do.  It's not a huge deal, but if other consumers copy & paste and we discover something we missed or need to change, it'd be nice to change one spot.

Reply:
I like this so far. I'll try, maybe add classes as another parameter.

Reply:
This will be dealt with when `link` is added to core. So this is resolved.